src/liquid/components/ld-notification/test/ld-notification.e2e.ts

```diff
@@ -0,0 +1,252 @@
+            detail: {
+              content: 'I am an info message.',
+              type: 'info',
+            },
+          })
+        )
+      })
+
+      await page.waitForChanges()
+
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders placed at the bottom with prop placement set to "bottom"', async () => {
+      const page = await getPageWithContent(
+        `<ld-notification placement="bottom"></ld-notification>`
+      )
+
+      page.evaluate(() => {
+        window.dispatchEvent(
+          new CustomEvent('ldNotificationAdd', {
+            detail: {
+              content: 'I am an info message.',
+              type: 'info',
+            },
+          })
+        )
+      })
+
+      await page.waitForChanges()
+
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+  })
+
+  describe('notification types', () => {
+    it('renders a notification of type "info"', async () => {
+      const page = await getPageWithContent(
+        `<ld-notification></ld-notification>`
+      )
+
+      page.evaluate(() => {
+        window.dispatchEvent(
+          new CustomEvent('ldNotificationAdd', {
+            detail: {
+              content: 'I am an info message.',
+              type: 'info',
+            },
+          })
+        )
+      })
+
+      await page.waitForChanges()
+
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders a notification of type "warn"', async () => {
+      const page = await getPageWithContent(
+        `<ld-notification></ld-notification>`
+      )
+
+      page.evaluate(() => {
+        window.dispatchEvent(
+          new CustomEvent('ldNotificationAdd', {
+            detail: {
+              content: 'I am a warning.',
+              type: 'warn',
+            },
+          })
+        )
+      })
+
+      await page.waitForChanges()
+
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+
+    it('renders a notification of type "alert"', async () => {
+      const page = await getPageWithContent(
+        `<ld-notification></ld-notification>`
+      )
+
+      page.evaluate(() => {
+        window.dispatchEvent(
+          new CustomEvent('ldNotificationAdd', {
+            detail: {
+              content: 'Oooops.',
+              type: 'alert',
+            },
+          })
+        )
+      })
+
+      await page.waitForChanges()
+
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+  })
+
+  describe('notification hierarchy', () => {
+    it('queues potentially less important notifications behind notifications of type "alert"', async () => {
```

Comment:
Instead of re-testing the order behavior, which you already did in the unit tests, I'd rather test how different types of notifications render on top with other notifications below and especially with 2, 3, 4 and 5 messages in the stack.